data/categories.txt

```diff
@@ -1444,7 +1444,7 @@ uk:^Комп'ютерний магазин|комп'ютерна техніка|
 vi:^Cửa hàng máy tính
 zh-Hans:^电脑商店|商店
 zh-Hant:^電腦商店|電腦專賣店|購物
-el:^Κατάστημα υπολογιστών|κατάστημα
+el:^Κατάστημα πληροφορικής|κατάστημα
```

Comment:
Feel free to remove general terms that are already mentioned in group categories.
For example,  ```κατάστημα``` here will be taken from ```@shop```.